addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComRfyMessage.java

```diff
@@ -95,27 +92,18 @@ public static SubType fromByte(int input) throws RFXComUnsupportedValueException
         }
     }
 
-    private static final List<RFXComValueSelector> SUPPORTED_INPUT_VALUE_SELECTORS = Arrays
-            .asList(RFXComValueSelector.SIGNAL_LEVEL, RFXComValueSelector.COMMAND);
-
-    private static final List<RFXComValueSelector> SUPPORTED_OUTPUT_VALUE_SELECTORS = Arrays
-            .asList(RFXComValueSelector.SHUTTER, RFXComValueSelector.VENETIAN_BLIND);
-
     private boolean sunDetector;
```

Comment:
Unused variable

Reply:
@mjagdis I see that you introduced this, can you validate whether it should still be used?

Reply:
It seems to have been introduced in https://github.com/openhab/openhab2-addons/pull/2328. But even then it already was an unused variable.